mono/eglib/gstr.c

```diff
@@ -1128,3 +1128,35 @@ g_strnlen (const char* s, gsize n)
 	for (i = 0; i < n && s [i]; ++i) ;
 	return i;
 }
+
+/**
+ * Loads a chunk of data from the file pointed to by the
+ * @fd starting at the file offset @offset for @size bytes
+ * and returns an allocated version of that string, or NULL
+ * on error.
+ */
+char *
+g_str_from_region (gint fd, guint64 offset, guint64 size)
```

Comment:
Suggest `read` or `file` or `fd` be in function name?
`g_read_string`?
`g_string_from_file`?
I not sure, if `read` implies `file`.
Also no point in `gint` over `int`.

Reply:
I'll just do `g_str_from_file_region`.